Networking/Services/UserManager.swift

```diff
@@ -200,7 +199,7 @@ public extension SessionManager {
     
```

Comment:
logout() and expire() are doing the same 🤔 , is there a reason for this? or should we delete the expire func?

Reply:
they are doing the same, we can delete one of them. Nevertheless I don't think this class is doing anything worth mantaining, I would delete it in the next release. Or maybe move it to Utils or something.

Reply:
I don't see a real need for this class, besides maybe saving the token but its trivial. I just decoupled it from the rest of the framework, but didn't modify anything of its funcitonality.
I don't think its worth it to refactor or do anything with this, but we can

Reply:
I guess adding it to Utils would be the best option, but we can talk about it in the next monthly iOS meeting